tests/.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
```

Comment:
What is this comment for?

Reply:
As I write [above](https://github.com/bratislava/bratislava.sk/pull/1182#discussion_r1504610642).